packages/celotool/src/cmds/deploy/destroy/blockscout.ts

```diff
@@ -13,11 +13,9 @@ export const builder = {}
 export const handler = async (argv: DestroyArgv) => {
   await switchToClusterFromEnv()
 
-  const instanceName = `${argv.celoEnv}${fetchEnvOrFallback('BLOCKSCOUT_DB_SUFFIX', '')}`
-  const helmReleaseName = `${argv.celoEnv}-blockscout${fetchEnvOrFallback(
-    'BLOCKSCOUT_DB_SUFFIX',
-    ''
-  )}`
+  const dbSuffix = fetchEnvOrFallback('BLOCKSCOUT_DB_SUFFIX', '')
+  const instanceName = `${argv.celoEnv}${dbSuffix}`
+  const helmReleaseName = `${argv.celoEnv}-blockscout${dbSuffix}`
```

Comment:
Seems like in the destroy, initial, and upgrade command we're creating the same strings. Could you add the common code where the helm release name is generated into a shared function? That way future changes will be easier